Content.Server/MachineLinking/Events/SignalValueRequestedEvent.cs

```diff
@@ -1,19 +0,0 @@
-using System;
-using Robust.Shared.GameObjects;
-
-namespace Content.Server.MachineLinking.Events
-{
-    public sealed class SignalValueRequestedEvent : HandledEntityEventArgs
-    {
-        public readonly string Port;
-        public readonly Type Type;
-
-        public object? Signal;
-
-        public SignalValueRequestedEvent(string port, Type type)
-        {
-            Port = port;
-            Type = type;
-        }
-    }
```

Comment:
I'm not sure if it would be used much, but it seems like you could trivially keep the "value-request" functionality by just having the `SignalTransmitterPort` class store the most recently transmitted value.